bin/get-specs

```diff
@@ -29,5 +29,7 @@ Nokogiri::HTML(get_url('aws-template-resource-type-ref.html')).css('.highlights
     CF
   end
 
-  File.write(File.join('specs', aws_name.scan(/AWS::(.*?)::(.*?)(?:\z|")/).first.join('-') + '.cf'), specs)
+  scanned = aws_name.scan(/AWS::(.*?)::(.*?)(?:\z|")/).first
+  scanned = %w[CloudFormation CustomResource] if scanned.nil?
```

Comment:
This line because AWS decided to make custom resources not have "AWS::" as the prefix.